ceph-extra-exporter/README.md

```diff
@@ -3,6 +3,33 @@ ceph-extra-exporter
 
 `ceph-extra-exporter` exposes extra metrics using some Ceph commands.
 
+## How to work
+
+ceph-extra-exporter get metrics using `ceph`, `rados-gw-admin`, and related commands and export metric values like as follows.
+
+- Run `ceph` commands etc written in the code.
+- Modify output JSON to specify format using jq filter.
+- Decode formatted JSON and export these values as the metrics.
+
+The commands to get values and arguments for jq are hard-coded, so to add metrics, please modify the program by reading [`How to add metrics`](#how-to-add-a-metrics) section.
```

Comment:
Though it is not a problem of this PR, "how-to-add-a-metrics" is a broken English.
It should be "how-to-add-metrics" or "how-to-add-a-metric", and the name of the related section should be also changed.